src/client/csfle/options.rs

```diff
@@ -20,12 +19,10 @@ use crate::{
 /// https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/client-side-encryption.rst#libmongocrypt-auto-encryption-allow-list
 /// )). To bypass automatic encryption for all operations, set bypassAutoEncryption=true in
 /// AutoEncryptionOpts.
-#[derive(Debug, TypedBuilder, Clone)]
+#[derive(Debug, Clone)]
 #[non_exhaustive]
-#[builder(field_defaults(setter(into)))]
```

Comment:
Since there's still a public `Client::with_encryption_options`, I think we probably want to retain the builder functionality here.

Reply:
Hmm.  Is it worth keeping `Client::with_encryption_options`, though?